def raise_output_exists(out):
    raise ValueError("output {out} exists!")
